src/ripple/app/tx/impl/Change.cpp

```diff
@@ -329,6 +329,15 @@ Change::applyAmendment()
                              << " activated: server blocked.";
             ctx_.app.getOPs().setAmendmentBlocked();
         }
+
+        if (view().rules().enabled(featureAmendmentTableSequence) ||
```

Comment:
It might be nice to increment the serial number when amendments gain or loose majority too since those change the amendment ledger entry.